ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -628,10 +746,18 @@ where is_in_physics_bubble(vessel)
 #endif
 
   private void ReferenceFrameSelection() {
+    bool used_to_fix_nav_ball_in_plotting_frame =
```

Comment:
"used_to_" doesn't read well in variable name, you wonder if it says that "this bool is used to...".  I suggest "was_fixing_".